### ISSUE TITLE
[Feature Request]: Create a planet.getpublii.com

### ISSUE THREAD

**Issue by s3nnet** [GetPublii/Publii#1834]
### Feature Description

With all blogs (partially) dedicated to Publii. Accept my application!

**Comment by bttrx** [comment]
There already is https://getpublii.com/showcase/

**Comment by s3nnet** [comment]
Not really. A Planet (think of Planet KDE) is an open blog network with posts of supporters and contributors about aal sorts of stuff.

**Comment by bttrx** [comment]
> think of Planet KDE

Never heard. So, thanks for bringing this to my attention.
